src/wallet/storage.rs

```diff
@@ -37,10 +63,12 @@ pub struct WalletStore {
     pub(super) prevout_to_contract_map: HashMap<OutPoint, ScriptBuf>,
     /// Map for all the fidelity bond information. (index, (Bond, script_pubkey, is_spent)).
     pub(super) fidelity_bond: HashMap<u32, (FidelityBond, ScriptBuf, bool)>,
-    //TODO: Add last synced height and Wallet birthday.
+    /// last synced height
     pub(super) last_synced_height: Option<u64>,
-
+    /// Wallet birthday
     pub(super) wallet_birthday: Option<u64>,
+    /// list of unspent transaction output
+    pub(super) list_unspent: Mutex<Vec<ListUnspentInfo>>,
```

Comment:
A RwLock could be more efficient than Mutex as we don't need to block at all read locks. RwLock allows multiple reads, but only one write lock.